libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -679,6 +679,10 @@ void AC_PosControl::update_xy_controller()
     // add velocity feed-forward scaled to compensate for optical flow measurement induced EKF noise
     vel_target *= ahrsControlScaleXY;
 
+    if (_xy_control_scale_factor > 0) {
```

Comment:
Why was this added?

Reply:
@lthall 
@tridge added this to be able to scale the velocity and position inputs of the position controller while using System ID feature in a position controlled mode.  Currently he set the scaler to 0.1 to reduce the position and velocity feedback while system ID is done.

Reply:
Cool.
We should change this to match the pattern in Attitude Control where we set it to 1.0 by default.

If this is a "Control" scale factor we should also update the accel_target like we do with the ahrsControlScaleXY.
accel_target *= ahrsControlScaleXY;

Reply:
i've changed it to default to 1 and removed the > 0 checks, it already did the accel_target